### PR TITLE
Sync GitHub Issues

### PR DESCRIPTION
**Pull request by adrianabedon**

### CODE REVIEW

**Review by daviddavis — APPROVED** [review]
One suggestion but overall this change looks great. Thanks for working on it!

**Review by mbearup — APPROVED** [review]

File: .github/scripts/issue_data.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+query_date = datetime.datetime.now(datetime.UTC) - datetime.timedelta(hours=1)
+
+unique_comment_authors = set()
+unique_reaction_authors = set()
+mentions_count = 0
+
+for comment in issue.get_comments(query_date):
+    if comment.user and comment.user.login not in unique_comment_authors:
+        unique_comment_authors.add(comment.user.login)
```
[the remark this answers is not in the document]
Comment (adrianabedon) [reply .github/scripts/issue_data.py]: Changed it!

File: .github/scripts/issue_data.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    if (
+        timeline_event.event in ["cross-referenced", "referenced"]
+        and timeline_event.created_at > query_date
+    ):
+        mentions_count += 1
+
+for reaction in issue.get_reactions():
+    if reaction.user:
+        if reaction.created_at > query_date:
+            unique_reaction_authors.add(reaction.user.login)
```
Comment (jasonzio) [inline .github/scripts/issue_data.py]: `unique_reaction_authors = {reaction.user.login for reaction in issue.get_reactions() if reaction.user and reaction.created_at > query_date}`

Comment (adrianabedon) [reply .github/scripts/issue_data.py]: Did this in the commit just before you comment.

File: .github/scripts/issue_data.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+for comment in issue.get_comments(query_date):
+    if comment.user and comment.user.login not in unique_comment_authors:
+        unique_comment_authors.add(comment.user.login)
+
+for timeline_event in issue.get_timeline():
+    if (
+        timeline_event.event in ["cross-referenced", "referenced"]
+        and timeline_event.created_at > query_date
+    ):
+        mentions_count += 1
```
Comment (jasonzio) [inline .github/scripts/issue_data.py:44]:
```
mentions_count = sum(1 for timeline_event in issue.get_timeline() if timeline_event.event in ["cross-referenced", "referenced"] and timeline_event.created_at > query_date)
```
feels awfully code-golf-ish.

Also idiomatic:
```
mentions_count = sum(map(lambda e: e.event in ["cross-referenced", "referenced"] and e.created_at > query_date, issue.get_timeline()))
```

Comment (adrianabedon) [reply .github/scripts/issue_data.py:44]: Changed

**Review by jasonzio — APPROVED** [review]
lgtm

**Review by mbearup — APPROVED** [review]

**Pull request merged**